### PR TITLE
Update submodule

### PR DESCRIPTION
**Pull request by ShanaLMoore**
- [ ] updates submodule to pull in bug fixes from hyku (sha: f389eac4)
- [ ] updates Gemfile.lock based off of submodule changes. Then ran bundle to include bundler.d specifications.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,1581 @@
...
+      parallel (>= 0.7.1)
+      public_suffix (>= 2)
+      rack (>= 1.3.6)
+
+GIT
+  remote: https://github.com/scientist-softserv/iiif_print.git
+  revision: 805191e6caedf2b5b91eb4e0aad692d4aff871c1
+  branch: main
+  specs:
+    iiif_print (2.0.0)
```
Comment (laritakr) [inline Gemfile.lock:218]: Shouldn't we update to 2.0.1 to bring in the bug fixes?

Comment (ShanaLMoore) [reply Gemfile.lock:218]: Yep. I'll update hyku and redo this then.

Comment (ShanaLMoore) [reply Gemfile.lock:218]: @laritakr https://github.com/samvera/hyku/pull/2243

**Review by laritakr — APPROVED** [review]

**Comment by ShanaLMoore** [comment]
flaky spec, passes locally: rspec ./spec/features/advanced_search_spec.rb:17

**Pull request merged**